src/content/blog/upsun-thank-you-new-sponsors-needed.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Thanks to Upsun, and Your Help is Needed Now!"
+pubDate: 2025-12-08
+summary: A thank-you to Upsun for years of support and ongoing support, but recognition that we need new sponsors after their support level decreases again next month.
+featureImage:
+  src: /img/blog/2025/12/upsun-ddev.png
+  alt: Upsun and DDEV Logos
+author: Randy Fay
+categories:
+  - Community
+---
+
+**TL;DR** Upsun has been (and remains) a great sponsor of DDEV, but is lowering their support level next month. We need you to fill the gap!
+
+I'm sure you remember how much Upsun (formerly Platform.sh) has helped DDEV along the way. When we were about to lose the ability to use the "DDEV" name, they stepped in and solved the problem. Then they [sponsored DDEV](platform-sh-becomes-a-lead-sponsor-of-ddev.md) at a very high level for a couple of years, and in 2025 at a [lower level](platform-sh-ddev-funding-changes.md). They have also managed the DDEV trademark and are now transferring it to the DDEV Foundation. We appreciate their significant ongoing support so very much, and honor their critical role in our history.
```

Comment:
Change "has helped DDEV along the way" to "has helped the DDEV project along the way"?

Also - change "the DDEV Foundation" to "the rapidly maturing DDEV Foundation"?

Change "We appreciate their significant ongoing support so very much, and honor their critical role in our history." to "We appreciate their significant ongoing support very much, and fully recognize and appreciate their critical role in our history."?